### DISCUSSION TITLE
Video/audio encryption

### DISCUSSION THREAD

**Discussion by ppamorim** [FreeTubeApp/FreeTube#2107] [General]
I have been thinking about this for years... We could use YouTube to just host a encrypted/masked video/audio. 

At first I thought to color shift the pixels from the video or even have multiple video sources to feed a single output.

Compression could be a problem, but I think multiple sources could help. The same could be done with audio.

**Comment by PrestonN** [comment]
This sounds like something outside of the scope of FreeTube. We'd have to create some sort of standard to make this work, and even if we did, convince others to use it. It isn't something that myself, and likely others on the team, would be interested in doing. This would be a new project for a new team.
